cereal/log.capnp

```diff
@@ -855,6 +855,9 @@ struct LiveLocationKalman {
   gpsTimeOfWeek @14 :Float64;
   status @15 :Status;
   unixTimestampMillis @16 :Int64;
+  inputsOK @17 :Bool = true;
```

Comment:
Hmm, why do these have initializers that set OK to true? If you forget to set it, it should default to bad. Does this matter on old data or something?

Reply:
yes it's for old data. Otherwise controlsd will think the sensors are offline on old liveLocationData in the process replay test. Once we have a better infrastructure to regenerate logs for testing we can take it out.